diff -urN cppunit-1.12.1-orig/include/cppunit/extensions/ExceptionTestCaseDecorator.h cppunit-1.12.1/include/cppunit/extensions/ExceptionTestCaseDecorator.h
--- cppunit-1.12.1-orig/include/cppunit/extensions/ExceptionTestCaseDecorator.h	2003-03-15 09:55:28.000000000 +0100
+++ cppunit-1.12.1/include/cppunit/extensions/ExceptionTestCaseDecorator.h	2013-11-29 19:17:25.780623386 +0100
@@ -92,7 +92,7 @@
    *
    * Should be overriden to check the exception.
    */
-  virtual void checkException( ExpectedExceptionType &e )
+  virtual void checkException( ExpectedExceptionType & /*e*/ )
   {
   }
 };
diff -urN cppunit-1.12.1-orig/include/cppunit/extensions/TestFixtureFactory.h cppunit-1.12.1/include/cppunit/extensions/TestFixtureFactory.h
--- cppunit-1.12.1-orig/include/cppunit/extensions/TestFixtureFactory.h	2007-01-12 05:54:34.000000000 +0100
+++ cppunit-1.12.1/include/cppunit/extensions/TestFixtureFactory.h	2013-11-29 19:17:25.780623386 +0100
@@ -30,6 +30,10 @@
 template<class TestFixtureType>
 class ConcretTestFixtureFactory : public CPPUNIT_NS::TestFixtureFactory
 {
+ public:
+  //! Standard destructor (necessary for gcc 41)
+  virtual ~ConcretTestFixtureFactory() {}
+
   /*! \brief Returns a new TestFixture instance.
    * \return A new fixture instance. The fixture instance is returned by
    *         the TestFixtureFactory passed on construction. The actual type 
